applications/asset_tracker_v2/src/cloud/lwm2m_integration/config_object_descript.xml

```diff
@@ -59,16 +59,36 @@
                 <Description><![CDATA[Number of seconds between each sampling/publication in passive mode, whether the device is moving or not.]]></Description>
             </Item>
             <Item ID="5">
-                <Name>Accelerometer threshold</Name>
+                <Name>Accelerometer activity threshold</Name>
                 <Operations>RW</Operations>
                 <MultipleInstances>Single</MultipleInstances>
                 <Mandatory>Mandatory</Mandatory>
                 <Type>Float</Type>
                 <RangeEnumeration></RangeEnumeration>
                 <Units></Units>
-                <Description><![CDATA[Accelerometer threshold in m/s². Minimal absolute value in m/s² for accelerometer readings to be considered valid movement.]]></Description>
+                <Description><![CDATA[Accelerometer activity threshold in m/s². Minimum absolute value in m/s² for accelerometer readings to be considered valid movement.]]></Description>
             </Item>
             <Item ID="6">
+                <Name>Accelerometer inactivity threshold</Name>
```

Comment:
Append to the list instead of creating new entries in the middle to avoid backwards compatibility issues (like for example, with the new schema uploaded to AV Systems, the user may intend to change the accelerometer inactivity timeout (with RID =7) but end up changing Neighbor cell measurements enable (with old RID = 7) in devices that are not yet updated). Remember to change the Object IDs in lwm2m_code.c as well when you change this. 

Also as per @simensrostad , there should be a note in the changelog which tells users to upload this new schema to AV Systems if they want to use the new parameters.